tests/symbolicator/snapshots/SymbolicatorUnrealIntegrationTest/test_unreal_apple_crash_with_attachments.pysnap

```diff
@@ -89,7 +89,6 @@ contexts:
     misc_cpu_vendor: GenuineIntel
     misc_number_of_cores: 4
     misc_number_of_cores_inc_hyperthread: 8
-    misc_primary_gpu_brand: Radeon Pro 560
```

Comment:
https://github.com/getsentry/relay/pull/650 👀

Reply:
yes

Reply:
@untitaker how come this didn't read `misc_primary_cpu_brand`? I thought we always pop `misc_primary_gpu_brand` and left the `cpu` one in.